reascripts/ReaSpeech/source/CurlRequest.lua

```diff
@@ -110,7 +110,7 @@ function CurlRequest:ready()
     return false
   end
 
-  if app:trap(function()
+  if pcall(function()
```

Comment:
Since we now need to hit an API endpoint at initialization time, the `app` global doesn't exist yet. Using `pcall` throws away the JSON syntax error info, but in practice if the JSON is invalid, it's probably because it's either not JSON at all, or the response was truncated. Either way, since JSON is constructed automatically, the exact syntax error isn't really actionable information.